api/v3/Case.php

```diff
@@ -218,16 +218,18 @@ function civicrm_api3_case_get($params) {
 
   // Add clause to search by client
   if (!empty($params['contact_id'])) {
-    $contacts = array();
-    foreach ((array) $params['contact_id'] as $c) {
-      if (!CRM_Utils_Rule::positiveInteger($c)) {
-        throw new API_Exception('Invalid parameter: contact_id. Must provide numeric value(s).');
+    // Convert from nonstandard format
```

Comment:
I think it would be better to state what the non standard form is & the fact you are converting for legacy compatibility